### ISSUE TITLE
Whisper-large instead of whisper small?

### ISSUE THREAD

**Issue by sleepingcat4** [Plachtaa/seed-vc#131]
I read the code and it is very clear if I change Whsiper-small to Whisper larger, what output dim, I should change? @Plachtaa do you have any hints or directions?

**Comment by sleepingcat4** [comment]
thanks for the quick answer! btw if I want to increase parameters to upto 1B, what changes to the DiT architecture should be made? do you have any advice

**Comment by Plachtaa** [comment]
I don't really suggest you to do so as the merit of VC model should be real-time and lightweight, it is not a difficult task that it worth's scaling up to 1B

**Comment by sleepingcat4** [comment]
@Plachtaa I wanted to experiment and see if how it may behave since I had some spare compute. I was thinking to increase the number of hidden dim of DiT but if you could suggest some advice for experimentation only, it would be nice.

**Comment by Plachtaa** [comment]
for your reference

![Image](https://github.com/user-attachments/assets/ff355f13-d743-4e6e-89bc-f2d643482c7f)

**Comment by sleepingcat4** [comment]
@Plachtaa thank you for being so helpful. another question, if I change this voice encoder model from "nvidia/bigvgan_v2_22khz_80band_256x" to "https://huggingface.co/nvidia/bigvgan_v2_44khz_128band_512x" what param I should change in the config?

**Comment by Plachtaa** [comment]
see https://github.com/Plachtaa/seed-vc/blob/97544fff2a57db718424c13ecd77f415bd2c7d4d/configs/presets/config_dit_mel_seed_uvit_whisper_base_f0_44k.yml#L14

**Comment by GUUser91** [comment]
@Plachtaa and @sleepingcat4
I need help. I followed Plachtaa's instructions and edit the config_dit_mel_seed_uvit_whisper_small_wavenet.yml config file file from 
```
in_channels: 768
``` 
to
```
in_channels: 1280 
```
and 
```
name: "openai/whisper-small" 
```
to 
```
name: "openai/whisper-large". 
```
I also tried changing it to 
```
name: "openai/whisper-large-v2" 
```
and 
```
name: "openai/whisper-large-v3" 
```
and 
```
name: "openai/whisper-large-v3-turbo". 
```
Then I try fine tuning
```
python train.py --config ./presets/config_dit_mel_seed_uvit_whisper_small_wavenet.yml --dataset-dir Cartoon-Voice --run-name Cartoon-Voice --batch-size 2 --max-steps 600 --max-epochs 1000 --save-every 600 --num-workers 0
```
And no matter what whisper large model I use, the output file always start speaking in gibberish. Is it because the dataset I gathered is a total of 3 minutes and 20 seconds?
https://vocaroo.com/1cTRV93JtsPQ

For reference, source audio
https://vocaroo.com/1jKAso7ZrC4C
reference audio
https://vocaroo.com/1jM7CIP8gROA

**Comment by Plachtaa** [comment]
@GUUser91 You must run pretrain on large scale dataset once you swap encoder from whipser-small to whipser-large-v3

**Comment by sleepingcat4** [comment]
@GUUser91 definitely, should gather more data. I trained on 2 hours and received awesome results!

**Comment by GUUser91** [comment]
@sleepingcat4 
Did you fine tune a model or did you train from scratch?

**Comment by sleepingcat4** [comment]
@GUUser91 I just fine-tuned

**Comment by GUUser91** [comment]
@sleepingcat4 How many steps did you fine tune the model for?

**Comment by sleepingcat4** [comment]
@GUUser91 400 steps

**Comment by Gonzaluigi** [comment]
I'm doing the same, but it doesn't detect any GPU to train. It's training too slow in CPU, but I have GPU (rtx 4060)

**Comment by leminhnguyen** [comment]
hi @sleepingcat4, did you change whisper-small to whisper-large-v3 and finetuned only with 4 hours of data ?

**Comment by sleepingcat4** [comment]
@leminhnguyen Yes, I trained on a multiple speaker dataset did almost 4 hours of data. my data was in very high-quality, it's an 11labs dataset that I had developed and open-sourced a few days earlier on HF under my lab **Sleeping AI**.

**Comment by leminhnguyen** [comment]
@sleepingcat4 thanks friend, keep your good jobs!

**Comment by sleepingcat4** [comment]
For reference this is my notebook: https://colab.research.google.com/drive/1HeJgMIRpEMd87z5oAcfBfS8_YRLvrwr9?usp=sharing

**Comment by leminhnguyen** [comment]
@sleepingcat4  see that you are cloning the forked repo, and there is a bug related to loading the pretrained model in reference.py, which was fixed in https://github.com/Plachtaa/seed-vc/pull/140, let's check it.

**Comment by GUUser91** [comment]
@sleepingcat4 how come this model https://huggingface.co/sleeping-ai/11labs-seed worked for inference but https://huggingface.co/sleeping-ai/11labs-seed-large and https://huggingface.co/sleeping-ai/11labs-512-large produced unintelligible speech for inference? I've did the setup correctly after downloading your 11labs-512-large files
```
python app_svc.py --checkpoint ft_model.pth --config config_dit_mel_seed_uvit_whisper_base_f0_44k.yml --fp16 True
```
